README.md

```diff
@@ -1,29 +1,72 @@
-# simplex-messaging
+# SimpleX chat
 
-[![GitHub build](https://github.com/simplex-chat/simplex-messaging/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-messaging/actions?query=workflow%3Abuild)
-[![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplex-messaging)](https://github.com/simplex-chat/simplex-messaging/releases)
+[![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
+[![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplex-chat)](https://github.com/simplex-chat/simplex-chat/releases)
 
 ## Federated chat - private, secure, decentralised
 
 See [simplex.chat](https://simplex.chat) website for chat demo and the explanations of the system and how SMP protocol works.
 
-SMP protocol is semi-formally defined [here](https://github.com/simplex-chat/protocol).
+SimpleX chat prototype is a "thin" terminal UI on top of [SimpleXMQ](https://github.com/simplex-chat/simplexmq) message broker, that uses [SMP protocol](https://github.com/simplex-chat/simplexmq/protocol) and SMP agent protocol.
 
-Currently only these features are available:
-- simple 1-to-1 chat with multiple people in the same terminal window.
+These features are implemented:
+- 1-to-1 chat with multiple people in the same terminal window.
 - auto-populated recipient name - just type your messages.
-- default server is available to play with - `smp.simplex.im:5223` - and you can deploy your own (`smp-server` executable in this repo).
-- no global identity or names visible to the server(s) - for the privacy of contacts and conversations.
-- E2E encryption, with public key that has to be passed out-of-band (see below)
-- authentication of each command/message with automatically generated RSA key pairs, separate for each conversation, the keys are not used as identity (2048 bit keys are used, it can be changed in [code via rsaKeySize setting](https://github.com/simplex-chat/simplex-messaging/blob/master/apps/dog-food/Main.hs))
+- default server is available to use - `smp1.simplex.im:5223` - you can deploy your own server (`smp-server` executable in [simplexmq](https://github.com/simplex-chat/simplexmq) repo).
+- no global identity or any names visible to the server(s) - for the full privacy of your contacts and conversations.
+- E2E encryption, with RSA public key that has to be passed out-of-band (see below).
+- message signing and verification with automatically generated RSA keys.
+- message integrity validation (via including the digests of the previous messages).
+- authentication of each command/message with automatically generated RSA key pairs.
```

Comment:
rephrase "authentication of each command **at SimpleXMQ broker**"? otherwise it gets mixed up with "message signing and verification with automatically generated RSA keys"